This patch removes the custom output name option that is handled properly only
in the master branch of go-bindata-assetfs (see https://github.com/elazarl/go-bindata-assetfs/pull/32)
diff --git a/GNUmakefile b/GNUmakefile
index 2168ed8de..76bb2c3cd 100644
--- a/GNUmakefile
+++ b/GNUmakefile
@@ -345,7 +345,7 @@
 .PHONY: static-assets
 static-assets: ## Compile the static routes to serve alongside the API
 	@echo "--> Generating static assets"
-	@go-bindata-assetfs -pkg agent -prefix ui -modtime 1480000000 -tags ui -o bindata_assetfs.go ./ui/dist/...
+	@go-bindata-assetfs -pkg agent -prefix ui -modtime 1480000000 -tags ui ./ui/dist/...
 	@mv bindata_assetfs.go command/agent
 
 .PHONY: test-ui

